### PR TITLE
Add last release date, and last PR on main or master

### LINKED ISSUE #8: Add columns 'last release date' and 'is there a release Pull Request?'
Add a column 'is there a release Pull Request?' and provide the link, to help see 'modules being released'.

Add column 'last release date' to help see modules that need a new release

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add last release date
| Type?             |improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/ps-monitor-module-releases/issues/8
| How to test?      | 
| Possible impacts? | 

Screenshot - 2022-08-16T110320 617

### CODE REVIEW

File: src/BranchManager.php [hunk 1]
```diff
@@ -29,6 +30,14 @@ public function __construct(Client $client)
     public function getReleaseData($repositoryName)
     {
         $references = $this->client->api('gitData')->references()->branches('prestashop', $repositoryName);
+        
+        try{
```
Comment (matks) [inline src/BranchManager.php]:
```suggestion
        try {
```

File: src/BranchManager.php [hunk 2]
```diff
@@ -29,6 +30,14 @@ public function __construct(Client $client)
...
+            $release = $this->client->api('repo')->releases()->latest('prestashop', $repositoryName);
+            $date = new DateTime($release['created_at']);
+            $releaseDate = $date->format('Y-m-d H:i:s');
+        } catch(Exception $e){
```
Comment (matks) [inline src/BranchManager.php]:
```suggestion
        } catch(Exception $e) {
```

**Review by matks — COMMENTED** [review]
Looks good but little code style problem

Should we add php cs fixer to CI?

This repository is getting more commits I initially expected 😅 so maybe we need to introduce quality tools (PHP CS Fixer, PHPStan and a minimum set of tests)

**Comment by ghost** [comment]
_Should we add php cs fixer to CI?_

Good idea.

Otherwise I added a link to the last PR in main or master but contriburor don't alway use the good branch :(

**Comment by PrestaEdit** [comment]
> This repository is getting more commits I initially expected 😅 

Welcome to the Open-Source 😌

**Comment by matks** [comment]
Tested on localhost ✅

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @okom3pom

**Pull request merged**